packit_service/config.py

```diff
@@ -272,17 +272,26 @@ def get_project_to_sync(self, dg_repo_name, dg_branch) -> Optional[ProjectToSync
 class PackageConfigGetter:
     @staticmethod
     def create_issue_if_needed(
-        project: GitProject, title: str, message: str
+        project: GitProject,
+        title: str,
+        message: str,
+        comment_to_existing: Optional[str] = None,
```

Comment:
I have only seen it used as `boolean` parameter, do you have any plans in mind with the `str`? ;)

Reply:
It's used as a test for the comment. I wanted to differentiate the text used for issue description and issue comment.

Reply:
@mfocko OK, I've removed the contact part from the comment since it will be in the description. (And now, it makes sense to have it different..;-)